From 893c4992a30523403ce9a707dfc81d8df81be2f1 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Tue, 19 Mar 2019 18:36:21 +0100
Subject: [PATCH 1/1] efi_loader: missing return in
 efi_get_next_variable_name()

Add a missing return statement in efi_get_next_variable_name().

Fixes Coverity CID 185834

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 lib/efi_loader/efi_variable.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/efi_loader/efi_variable.c b/lib/efi_loader/efi_variable.c
index e0d7f5736db..699f4184d93 100644
--- a/lib/efi_loader/efi_variable.c
+++ b/lib/efi_loader/efi_variable.c
@@ -335,7 +335,7 @@ efi_status_t EFIAPI efi_get_next_variable_name(efi_uintn_t *variable_name_size,
 	EFI_ENTRY("%p \"%ls\" %pUl", variable_name_size, variable_name, vendor);
 
 	if (!variable_name_size || !variable_name || !vendor)
-		EFI_EXIT(EFI_INVALID_PARAMETER);
+		return EFI_EXIT(EFI_INVALID_PARAMETER);
 
 	if (variable_name[0]) {
 		/* check null-terminated string */
-- 
2.20.1

